blog/2022-03-07-update_all_the_things.md

```diff
@@ -0,0 +1,17 @@
+---
+author: Joakim Sørensen
+authorURL: https://github.com/ludeeus
+authorImageURL: /img/profile/ludeeus.jpg
+authorTwitter: ludeeus
+title: "Update all the things!🎉"
+---
+
+In Home Assistant Core [2021.12](https://www.home-assistant.io/blog/2021/12/11/release-202112/#brand-new-configuration-panel) we moved updates provided by
+the Supervisor to the Configuration panel, starting with Home Assistant Core 2022.4 this is no longer limited to the Supervisor integration.
+
+All integrations can now provide an update platform (`update.py`) to show and
+perform updates.
+
+Previously we had the `update` device class for the [button entity](/docs/core/entity/button), this is now considered deprecated and you should move to the update platform instead.
```

Comment:
I would not agree on that, as it is currently not possible to automate it?
The button can be automated.

Reply:
I have no opinion on this.

Reply:
Right, well duplicates don't help, so I kinda agree. Was more a conclusion towards: Should it be exposed a service to be able to trigger an update in some way?

Reply:
For feature requests please open a thread on our forum :rofl:

The update handler needs the domain, identifier, and version of the update that is being performed (which changes between updates), having that in a service would be hard for the user to maintain as no events or entities are created with that information

Reply:
In that case, we should revisit the platform and maybe consider creating entities IMHO.

Reply:
In that case, this PR makes no sense.